k8s/deployments/mariadb.yaml

```diff
@@ -33,26 +21,11 @@ spec:
                   name: mpm-secrets
                   key: DB_PASSWORD
             - name: MYSQL_DATABASE
-              value: micro_power_manager
+              value: database-1
+            - name: MYSQL_HOST
+              value: database-1.cli8wgsymz96.eu-central-1.rds.amazonaws.com
```

Comment:
Please don't put plain text links to our AWS account here. Could you move this to a secret?